lib/sentry/rescue_evss_errors.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Sentry
+  module RescueEVSSErrors
+    private
```

Comment:
i don't think this keyword does anything in a module, private methods can still be called publicly

Reply:
Not sure when they added it, but [it does work now](https://ruby-doc.org/core-2.3.0/Module.html#method-i-private).